openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryV1.component.tsx

```diff
@@ -477,249 +373,212 @@ const GlossaryV1 = ({
     }
   }, [selectedData, isGlossaryActive]);
 
-  return glossaryList.length ? (
-    <PageLayoutV1 leftPanel={fetchLeftPanel()}>
-      {isImportAction ? (
-        <ImportGlossary glossaryName={selectedData.name} />
-      ) : (
-        <>
-          <div
-            className="tw-flex tw-justify-between tw-items-center"
-            data-testid="header">
-            <div
-              className="tw-text-link tw-text-base"
-              data-testid="category-name">
-              <TitleBreadcrumb
-                titleLinks={breadcrumb}
-                widthDeductions={
-                  leftPanelWidth + addTermButtonWidth + manageButtonWidth + 20 // Additional deduction for margin on the right of leftPanel
-                }
-              />
-            </div>
-            <div
-              className="tw-relative tw-flex tw-justify-between tw-items-center"
-              id="add-term-button">
-              <Tooltip
-                title={
-                  createGlossaryTermPermission
-                    ? 'Add Term'
-                    : NO_PERMISSION_FOR_ACTION
-                }>
-                <ButtonAntd
-                  className="tw-h-8 tw-rounded tw-mr-2"
-                  data-testid="add-new-tag-button"
-                  disabled={!createGlossaryTermPermission}
-                  type="primary"
-                  onClick={handleAddGlossaryTermClick}>
-                  Add term
-                </ButtonAntd>
-              </Tooltip>
-
-              <Dropdown
-                align={{ targetOffset: [-12, 0] }}
+  return isImportAction ? (
+    <ImportGlossary glossaryName={selectedData.name} />
+  ) : (
+    <div>
+      <div
+        className="tw-flex tw-justify-between tw-items-center"
+        data-testid="header">
+        <div className="tw-text-link tw-text-base" data-testid="category-name">
+          <TitleBreadcrumb
+            titleLinks={breadcrumb}
+            widthDeductions={
+              leftPanelWidth + addTermButtonWidth + manageButtonWidth + 20 // Additional deduction for margin on the right of leftPanel
+            }
+          />
+        </div>
+        <div
+          className="tw-relative tw-flex tw-justify-between tw-items-center"
+          id="add-term-button">
+          <Tooltip
+            title={
+              createGlossaryTermPermission
+                ? 'Add Term'
```

Comment:
Please take care of the Localisation

Reply:
I will handle it in a separate patch thanks!